cluster/gce/gci/configure-helper.sh

```diff
@@ -752,10 +752,10 @@ function start-kube-apiserver {
   fi
   local -r src_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty"
 
-  if [[ -n "${KUBE_USER:-}" ]]; then
+  if [[ -n "${KUBE_USER}" ]]; then
```

Comment:
With nounset, `${THIS_IS_UNSET:-}` will not fail. [This is a super-useful resource](http://tldp.org/LDP/abs/html/parameter-substitution.html).